jme3-android-examples/jme3-android-tests/jbullet/build/.transforms/9133b346721647632b4fac78e5bfd092.bin

```diff
@@ -0,0 +1 @@
+i/
```

Comment:
what is this .bin file?

Reply:
I don't know what these files are really, but they are getting generated by Android Jetifier when importing jbullet & stack-alloc (as you know from the forum) to the main android module

![image](https://user-images.githubusercontent.com/60224159/114263388-c267ba80-99aa-11eb-995e-44fe5673a51d.png)